src/index.html

```diff
@@ -20,8 +20,307 @@
+        </div>
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </button>
+      </div>
+      <div class="card">
+        <div class="card__photo"></div>
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="stars">
+          <div class="stars__total stars__total--n">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="stars__number-of-reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price-block">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </button>
+      </div>
+      <div class="card">
+        <div class="card__photo"></div>
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="stars">
+          <div class="stars__total stars__total--n">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="stars__number-of-reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price-block">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </button>
+      </div>
+      <div class="card">
+        <div class="card__photo"></div>
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="stars">
+          <div class="stars__total stars__total--n">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="stars__number-of-reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price-block">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </button>
+      </div>
```

Comment:
All product cards contain the same details, including the product name, code, reviews, and price. If this is not intentional, consider using a template or a data-driven approach to populate these cards with unique product information.